public/static/js/admin.js

```diff
@@ -87,6 +88,28 @@ mciModule.controller('AdminSettingsController', ['$scope','$window', 'mciAdminRe
       return;
     }
 
+    try {
+      $scope.Settings.container_pools.pools = jsyaml.safeLoad($scope.tempContainerPools);
```

Comment:
After converting from yaml, I would do some basic validation that each pool object is in the format that we have in the model